Mathlib/Algebra/BigOperators/Expect.lean

```diff
@@ -98,7 +101,7 @@ to show the domain type when the expect is over `Finset.univ`. -/
         `(bigOpBinder| $(.mk i):ident)
     `(𝔼 $binder:bigOpBinder, $body)
   else
-    let ss ← withNaryArg 3 <| delab
+    let ss ← withNaryArg 4 <| delab
```

Comment:
What's this change doing? Or more specifically, how do I know there won't be n more PRs incrementing the number again :D

Reply:
Before this PR, `𝔼 i, f i` pretty-prints as `𝔼 i : Module ℚ≥0 M, f i`. After this PR, it will pretty-print as `𝔼 i : ι, f i`. I just originally mistakenly copied the argument numbers from the `Finset.sum` delaborator without tweaking them.